lib/topological_inventory/schema/default.rb

```diff
@@ -147,18 +147,19 @@ def service_instance_tasks_custom_save(source, tasks_collection)
         return if src_refs.blank?
 
         # Updating Tasks
-        # service_instance_tasks_update_by_raw_sql(source)
-        service_instance_tasks_update_by_activerecord(tasks_collection, source, src_refs)
+        service_instance_tasks_update_by_raw_sql(source)
```

Comment:
I think we can pull this change out of the PR.  I understand the reasoning for wanting to put it in, but it falls outside the changes specific to exposing the artifacts on a service_instance.

I  would be interested in moving to the raw sql method when there is a clear performance bottleneck we need to resolve.

Reply:
@syncrou whole refresh is built on raw sql, this is one of the few exceptions - https://github.com/ManageIQ/inventory_refresh/blob/master/lib/inventory_refresh/save_collection/saver/sql_helper_upsert.rb
Why do we need to wait for bottleneck and then solve it (and spend many hours with diagnostics probably on production)? Isn't it better to prevent it?
Ok, I'll remove it from this PR, but are you opened to move it to new PR?